src/core/core.layoutService.js

```diff
@@ -10,24 +10,58 @@ module.exports = function(Chart) {
 	Chart.layoutService = {
 		defaults: {},
 
-		// Register a box to a chartInstance. A box is simply a reference to an object that requires layout. eg. Scales, Legend, Plugins.
+		/**
+		 * Register a box to a chartInstance.
+		 * A box is simply a reference to an object that requires layout. eg. Scales, Legend, Title.
+		 * @method Chart.layoutService.addBox
```

Comment:
I did this documentation method based on http://stackoverflow.com/a/6460748 because it generated less docs than doing something to define an interface. Once we have `gulp docs` in, I can test this and see what it generates